src/transformers/keras_callbacks.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional
+
+from tensorflow.keras.callbacks import Callback
+
+
+class PushToHubCallback(Callback):
+    def __init__(
+        self,
+        save_strategy="epoch",
+        save_steps: Optional[int] = None,
+        repo_path_or_name: Optional[str] = None,
+        repo_url: Optional[str] = None,
+        **push_to_hub_kwargs
```

Comment:
Not sure we need to show those: the blocking argument should be set to False so training continues, and the commit messages should be different per step.

Also the tokenizer should be accepted as an optional argument.

Reply:
Done!